run_excel.py

```diff
@@ -0,0 +1,100 @@
+import os
+import logging
+import boto3
+import json
+import pandas as pd
+
+logger = logging.getLogger()
+logging.basicConfig()
+level = logging.getLevelName(os.getenv('LOG_LEVEL', 'INFO'))
+logger.setLevel(level)
+
+def main():
+    logger.info("Starting Security Hub Search")
+    Security_Hub_list = get_list_securityhub()
+    
+    logger.info("Saving Security Hub Search") 
+    df_json = pd.read_json(json.dumps(Security_Hub_list))
+    df_json.to_excel("Security Hub Findings.xlsx",index=False)
+    
+    logger.info("Security Hub Search Complete")
+
+def get_list_securityhub() -> list[str]:
+    """Gets all security hub items.
+    Returns:
+        list[str]: of filtered security hub finding according to the filter specified.
+    """
+    operation_filters = {'Filters': {
+            'SeverityLabel': 
+            [
+                {
+                    'Value': 'CRITICAL',
+                    'Comparison': 'EQUALS'
+                }
+            ],
+            'WorkflowStatus': 
+            [
+                {
+                    'Value': 'RESOLVED',
+                    'Comparison': 'NOT_EQUALS'
+                }#,
+                # {
+                #     'Value': 'SUPPRESSED',
+                #     'Comparison': 'EQUALS'
+                # }
+            ]
+            }
+        }
+    
+    response = paginate_results(client_type='securityhub', operation='get_findings', operation_parameters=operation_filters, response_key='Findings')
+    securityhub_ids = []
+    for finding in response:
+        
+        Id = finding['Id']
+        GeneratorId = finding['GeneratorId']
+        AwsAccountId = finding['AwsAccountId']
+        Title = finding['Title']
+        Description = finding['Description']
+        Severity = finding['Severity']['Label']
+        try:
+            Remediation_Text = finding['Remediation']['Recommendation']['Text']
```

Comment:
you don't need these except statements here, you can do `remediation_text = finding['Remediation']['Recommendation'].get('Text', '')`